setup.py

```diff
@@ -147,5 +147,7 @@ def run(self):
     entry_points={'console_scripts': [
         'prusalink = prusa.link.__main__:main',
         'prusalink-manager = prusa.link.multi_instance.__main__:main',
+        'prusalink-image-builder = prusa.link.image_builder.'
```

Comment:
I'm sure that `prusa-image-builder` is really not need to install with PrusaLink package.